static/js/components/dashboard/CourseListCard.js

```diff
@@ -45,6 +46,26 @@ export default class CourseListCard extends React.Component {
     checkout:                     (s: string) => void,
   };
 
+  getRegularPrice = (program: Program, prices: CalculatedPrices): Decimal|null => {
+    let price;
+    let list = [];
+
+    for (const courseRun of sortedCourseRuns(program)) {
+      price = prices.get(courseRun.id);
+      if (price) {
+        list.push(price);
+      }
+    }
+    // retruning max price, assuming that max price is regular price otherwise discounted.
+    // what happen when user has coupans for all course in program for COUPON_CONTENT_TYPE_COURSE.
```

Comment:
The way that coupons are structured currently, there should never be more than one coupon per program, even if they are for a course.